database/migrations/default/1741719786807_update_coord_partners_lookup/up.sql

```diff
@@ -0,0 +1,29 @@
+-- Updating table name to coordination_partners and moving to lookups schema
+alter table if exists atd__coordination_partners_lkp rename to coordination_partners;
+alter table if exists coordination_partners set schema lookups;
+alter table if exists lookups.coordination_partners rename column coord_partner_desc to label;
+alter table if exists lookups.coordination_partners add column source text default 'vz' not null;
+
+alter index lookups.atd__coordination_partners_lkp_pkey rename to coordination_partners_pkey;
+
+-- Cannot undo dropping the sequence in the down migration
+drop sequence lookups.atd__coordination_partners_lkp_id_seq cascade;
```

Comment:
i cant really undo this without things getting weird, i would have to create the sequence again in the down migration but instead have it start at 8 to account for the IDs that are already assigned. is it okay to leave it like this and not account for it in the down?